src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -0,0 +1,530 @@
+<%@ tag description="instructorFeedbacks - new feedback session form" %>
```

Comment:
I'm sure it was more ridiculous originally, but would it be worth it to split this further?
It's still pretty long at 500+ lines.